src/ga4gh/vrs/extras/vcf_annotation.py

```diff
@@ -0,0 +1,95 @@
+"""
+Annotate VCF files with VRS
+
+Input Format: VCF
+Output Format: VCF
+
+"""
+
+import argparse
+import hashlib
+import logging
+import sys
+import pickle
+import time
+import coloredlogs
+from biocommons.seqrepo import SeqRepo
+from pysam import VariantFile
+from ga4gh.core import ga4gh_identify
+from ga4gh.vrs import models
+from ga4gh.vrs.dataproxy import SeqRepoDataProxy
+from ga4gh.vrs.extras.translator import Translator
+
+dp = SeqRepoDataProxy(SeqRepo("/usr/local/share/seqrepo/latest"))
+tlr = Translator(data_proxy=dp)
+
+_logger = logging.getLogger()
+
+INFO_FIELD_ID = "VRS_Allele"
+vrs_data = {}
+
+
+def parse_args(argv):
+    ap = argparse.ArgumentParser()
+    ap.add_argument("VCF_IN")
+    ap.add_argument("--out", "-o", default="-")
+    ap.add_argument("--failure-probability", "-f", type=float, default=0)
```

Comment:
Unused?

Reply:
I dont know why this is showing up as unused. I definitely run the command with these args.

Reply:
For example: python3 src/ga4gh/vrs/extras/vcf_annotation.py ~/Downloads/testing_benchmark.vcf --out ~/Documents/Wagner_Lab/Projects/test_ouput.vcf.gz --vrs_file ~/Documents/Wagner_Lab/Projects/vrs_objects.pkl

Reply:
@wesleygoar is failure-probability also used? If so I think you can resolve this safely.